object-assign-es6/index.html

```diff
@@ -0,0 +1,46 @@
+---
+feature_name: ES6 Object.assign()
+chrome_version: 45
+feature_id: REPLACE_ME
+additional_head_content: <meta name="optional" value="optional">
+---
+
+<h3>Background</h3>
+<p><code>Object.assign()</code> copies the values (of all enumerable own properties) from one or more source objects to a target object. It is useful for merging objects or cloning them shallowly.</p>
```

Comment:
Can you put in something about how the target object is the first parameter, and that the target object is also used as the return value? The feature is somewhat nonintuitive in that regard, and I could imagine folks getting confused.